cognite/client/data_classes/data_modeling/data_types.py

```diff
@@ -106,6 +105,9 @@ def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None =
                 # to avoid breaking changes. When used as a read object, the `list` will always be present.
                 is_list=resource.get("list", False),
             )
+        elif type_ == "enum":
+            values = {key: EnumValue.load(value) for key, value in resource["values"].items()}
```

Comment:
Can skip a stack frame & an isinstance check:
```suggestion
            values = {key: EnumValue._load(value) for key, value in resource["values"].items()}
```